components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -0,0 +1,106 @@
+<chef-page
```

Comment:
This page is a bit too tall, when viewed on my 15" Mac. If you compare to the previous use in policy-add-members.component.html, that page fits without any vertical scrolling. This page seems to have extra vertical whitespace that could be removed so it does not scroll on a standard 15" Mac (looks to be about 766 pixels high).

Reply:
@msorens That top nav needed to be hidden.